runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/internal/k3po/ext/behavior/ZillaEngine.java

```diff
@@ -285,7 +285,8 @@ private ZillaScope newScope(
         int scopeIndex)
     {
         ZillaScope scope = new ZillaScope(config, labels, scopeIndex, this::lookupTargetIndex,
-                System::nanoTime, traceIds::incrementAndGet);
+                //System::nanoTime, traceIds::incrementAndGet);
+                () -> 0L, traceIds::incrementAndGet); // TODO
```

Comment:
Understand this will change when we have `zilla:timestamps "false"` generating no timestamps in the dump.

Reply:
reverted